src/routes/github/subscription-deferred-install/github-subscription-deferred-install-get.ts

```diff
@@ -0,0 +1,53 @@
+import { Request, Response } from "express";
+import { extractSubscriptionDeferredInstallPayload } from "services/subscription-deferred-install-service";
+import { hasAdminAccess } from "services/subscription-installation-service";
+
+const escapeHtml = (unsafe: string) => {
+	return unsafe
+		.replace(/&/g, "&amp;")
+		.replace(/</g, "&lt;")
+		.replace(/>/g, "&gt;")
+		.replace(/"/g, "&quot;")
+		.replace(/'/g, "&#039;");
+};
+
+export const GithubSubscriptionDeferredInstallGet = async (req: Request, res: Response) => {
+	const payload = await extractSubscriptionDeferredInstallPayload(req);
+	const { githubToken, installation } = res.locals;
+
+	if (!await hasAdminAccess(githubToken, installation.jiraHost, payload.gitHubInstallationId, req.log, payload.gitHubServerAppIdPk)) {
+		req.log.warn("Not an admin");
+		res.status(401).json({
```

Comment:
/nit
User will see a json text right? I am okay for a spike.

Reply:
Yeah